### ISSUE TITLE
`−12:00`, `+12:45`, `+13:00`, `+13:45`, `+14:00` process as incorrect time zones

### ISSUE THREAD

**Issue by Semigradsky** [marnusw/date-fns-tz#171]
There is a breaking change in `1.3.0`.

After https://github.com/marnusw/date-fns-tz/commit/4ae18197a85bcdacd6fd939f207be4861cf29900 these time zones process as incorrect.
cc @marnusw

Is it a bug or `date-fns-tz` will no more work with these time zones?

References: [`−12:00`](https://en.wikipedia.org/wiki/UTC%E2%88%9212:00), [`+12:45`](https://en.wikipedia.org/wiki/UTC%2B12:45), [`+13:00`](https://en.wikipedia.org/wiki/UTC%2B13:00), [`+13:45`](https://en.wikipedia.org/wiki/UTC%2B13:45), [`+14:00`](https://en.wikipedia.org/wiki/UTC%2B14:00)

**Comment by marnusw** [comment]
OK, I take your point that this is a breaking change. That said, if these are not valid time zones then this was actually a bug fix which is why it was included.

Can you provide references indicating that these are indeed valid time zones? Are they not capped at -11:59 and +12:00?

**Comment by Semigradsky** [comment]
I has the references that it is a "valid UTC offset" -  https://en.wikipedia.org/wiki/List_of_UTC_time_offsets, https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Also I see that this offsets works with new Temporal object:
```js
Temporal.TimeZone.from('-12:00') // okay
Temporal.TimeZone.from('-23:59') // okay
Temporal.TimeZone.from('-24:00') // Uncaught RangeError: Invalid time zone: -24:00

Temporal.TimeZone.from('+14:00') // okay
Temporal.TimeZone.from('+23:59') // okay
Temporal.TimeZone.from('+24:00') // Uncaught RangeError: Invalid time zone: +24:00
```
https://github.com/tc39/proposal-temporal/blob/47765713c747da13b094247053534b2d2fed0bdf/polyfill/lib/regex.mjs#L17

Would be nice to have the same limitations as future native implementation.

**Comment by marnusw** [comment]
Thank you @Semigradsky, we can certainly match what `Temporal` supports in your example above. It's an easy fix, so if you need it urgently and have the time please submit a PR, but I'll do it when I can.

**Comment by Semigradsky** [comment]
I will try 👍

**Comment by marnusw** [comment]
Released in `v1.3.2`.
